src/fastoad/notebooks/02_explanation/03_scipy.ipynb

```diff
@@ -0,0 +1,388 @@
+{
```

Comment:
<blockquote>As for the MDA, we have the same results as in the <a href="https://github.com/fast-aircraft-design/FAST-OAD/raw/cf8fa880a6d979a72ead2cc6fe2c5613b7bd453d/src/fastoad/notebooks/02_explanation/02_pure_python.ipynb" target="_blank">previous notebook</a>, <strong>which is a good sign</strong>.</blockquote><p>If we keep this part, I would prefer "as expected" here.</p><p> </p><blockquote>The issue however, is that as the aspect ratio varies so will the MTOW.</blockquote><p>--&gt; "The issue however is that MTOW will vary with aspect ratio."</p><p>(for sake of simplicity)</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Changed in a03fe54a